src/server.cpp

```diff
@@ -1489,11 +1489,8 @@ void Server::SendMovement(u16 peer_id)
 	Send(&pkt);
 }
 
-void Server::SendPlayerHPOrDie(PlayerSAO *playersao)
+void Server::ForceSendPlayerHPOrDie(PlayerSAO *playersao)
 {
-	if (!g_settings->getBool("enable_damage"))
-		return;
```

Comment:
this would then allow killing the player even when damage is disabled, shouldn't this be kept the old way?